fb.d.ts

```diff
@@ -0,0 +1,61 @@
+import Promise = require("any-promise");
```

Comment:
That's how i have seen it used in most well written typings I have used. If you are not targeting ES6, it is up to the consumer to provide the typings for `Promise`. Example from a fairy recent declaration https://github.com/facebook/dataloader/blob/master/src/index.d.ts